tools/devops/automation/build-pipeline.yml

```
@@ -98,6 +98,8 @@ stages:
         runTests: ${{ parameters.runTests }}
         runDeviceTests: ${{ parameters.runDeviceTests }}
         keyringPass: $(xma-password)
+        gitHubToken: ${{ variables['GitHub.Token'] }}
```

Why using ${{ }} over $() for a variable?
```suggestion
        gitHubToken: $(GitHub.Token)
```
Is the '.' right?

---

They should be semantically equivalent in this case.

I chose to use ${{ }} here because the value of GitHub.Token is known at compile-time, since ${{ }} gets replaced before the pipeline runs (where as $() gets replaced as the pipeline is running). It's become habit for me to use ${{ }} since working on GitHub actions because a lot of substitutions there can only be made during compile time.